Umbraco-Cloud/Troubleshooting/Deployments/Structure-Error/index.md

```diff
@@ -24,94 +20,124 @@ Example:
+
+In this example there are two `.uda` files who share the same alias which leads to a conflict: it is impossible for Deploy to know which of the files to use, so it gives up and sends an error back.
+
+:::note
+Does the collision error involve **Dictionary items**?
+Use this guide instead: [Troubleshooting duplicate dictionary items](../Duplicate-Dictionary-Items)
+:::
+
+You can run into an error like this on all of your Cloud environment. Somestimes you might also run into it, on a local clone of your project. This guide will use an example, where two files are colliding on a Development and a Live environment.
 
-In the example above there are two files in the `~/data/revision` folder which contain the same alias, for a Document Type. Each file has a different file name due to having a different unique key (UDI), but the Document Type they both contain is `home`. When Deploy tries to create the site structure from these files, it is inspecting each file to try to create a Document Type with the aliases specified.
+## Table of content
 
-In the case above, there are two files (Document Type representations) who share the same alias which leads to a conflict: it's impossible for Deploy to know which is the "correct" file, so it gives up and sends an error back.
+* [Cause](#cause)
+* [Video tutorial](#video-tutorial)
+* [Using the error message](#using-the-error-message)
+* [Deciding which file you want to use](#deciding-which-file-you-want-to-use)
+* [Getting your environments in sync](#getting-your-environments-in-sync)
 
-### Cause
+## Cause
 
 <iframe width="800" height="450" src="https://www.youtube.com/embed/pF5SUh30FKI?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
 
-The main cause of this problem is when a Document Type (or Media Type, Data Type, etc) is manually created in two environments using the same alias. Since each of the environments are isolated and do not know what the other one is doing until they are synchronized - creating a Document Type in each environment will actually create duplicate Document Types that are considered separate entities even though they look the same and share the same alias in the backoffice of both environments.
+The main cause of this problem is when an entity has been manually created in two or more environments, using the same alias. 
+
+Each environment on Umbraco Cloud are isolated and do not know what the other one is doing until they are syncronised. When an entity is created with identical aliases on two or more environments, it will create duplicate entities that are considered separate entities even though they look that same and share the same alias on both/all environments.
```

Comment:
```suggestion
Each environment on Umbraco Cloud are isolated and do not know what the other one is doing until they are synchronized. When an entity is created with identical aliases on two or more environments, it will create duplicate entities that are considered separate entities even though they look that same and share the same alias on both/all environments.
```